### PR TITLE
Update support files

### PR DESCRIPTION
**Pull request by metadoggo**
Based on feedback from @OliverJAsh and @theefer

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -15,9 +15,9 @@ module.exports = function(config) {
 
     // list of files / patterns to load in the browser
     files: [
-      'test-main.js',
+      'src/test-main.js',
```
Comment (theefer) [inline karma.conf.js]: Should that be `tests/test-main.js`?

Comment (metadoggo) [inline karma.conf.js]: oh yeah

File: .editorconfig [hunk 2]
```diff
@@ -3,7 +3,7 @@ root = true
 
 [*]
 indent_style = space
-indent_size = 2
+indent_size = 4
```
Comment (theefer) [inline .editorconfig:6]: Shouldn't the code also get reindented then?

Comment (metadoggo) [inline .editorconfig:6]: Done

**Comment by theefer** [comment]
:+1:

**Pull request merged**